### PR TITLE
Added translation services and GetMessageText function

### PR DESCRIPTION
**Pull request by mcrispino**

### CODE REVIEW

File: src/misc/getMessageText.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+/**
+ * Searches the language object (or the current language if that parameter is omitted) for a given message code
+ * @param {string} str The message identifier
+ * @param {string} languageName The language name, may be omitted
+ * @return {string} The translation of the given message in the specified language or the current language
+ */
+export const getMessageText = (str: string, languageName: string): string => {
+  const translator = TranslationService.getInstance();
+  if (!languageName || languageName.length === 0) {
```
Comment (ncardeli) [inline src/misc/getMessageText.ts]: An empty strings is evaluated as false
```suggestion
  if (!languageName) {
```

File: src/misc/getMessageText.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { TranslationService } from "../config/translationService";
+
+/**
+ * Searches the language object (or the current language if that parameter is omitted) for a given message code
+ * @param {string} str The message identifier
+ * @param {string} languageName The language name, may be omitted
```
Comment (ncardeli) [inline src/misc/getMessageText.ts]: What happens if omitted?

File: src/config/translationService.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  static getInstance() {
+    if (!TranslationService.instance) {
+      TranslationService.instance = new TranslationService();
+    }
+    return TranslationService.instance;
+  }
+
+  private constructor() {}
+
+  //
```
Comment (ncardeli) [inline src/config/translationService.ts]:
```suggestion
```

File: src/config/test/translationService.spec.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import { TranslationService, TranslationsData } from "../translationService";
```
Comment (ncardeli) [inline src/config/test/translationService.spec.ts]: 👍  for the tests
We should test `getImageSource` too

File: src/misc/getMessageText.ts (hunk 2 above)
Comment (mcrispino) [reply src/misc/getMessageText.ts]: That case is explained in the method's description :)

File: src/config/test/translationService.spec.ts (hunk 4 above)
Comment (mcrispino) [reply src/config/test/translationService.spec.ts]: Fixed @ [76b1d3c](https://github.com/genexuslabs/web-standard-functions/pull/65/commits/76b1d3c04b52ad84b43ada998a2365ab1408db1e)

**Pull request merged**